src/cljs/athens/keybindings.cljs

```diff
@@ -88,7 +88,8 @@
    ["Tomorrow"      mui-icons/Today (fn [] (str "[[" (:title (get-day -1)) "]]")) nil nil]
    ["Yesterday"     mui-icons/Today (fn [] (str "[[" (:title (get-day 1)) "]]")) nil nil]
    ["YouTube Embed" mui-icons/YouTube "{{[[youtube]]: }}" nil 2]
-   ["iframe Embed"  mui-icons/DesktopWindows "{{iframe: }}" nil 2]])
+   ["iframe Embed"  mui-icons/DesktopWindows "{{iframe: }}" nil 2]
+   ["Block Embed"   mui-icons/ViewDayRounded "{{[[embed]]: (())}}" nil 4]])
```

Comment:
A ton of edge cases. But I think it's best to catch these edge cases early because they are easy to fix if there is context. Else it will be hard to pick them up completely new at some other point. Keep adding more cases where you think it can start failing other than the usual suspects.   

Regarding parens -- is it an arbitrary number of matching parens(>2)? -- change should be simple enough

Reply:
Seems arbitrary

![image](https://user-images.githubusercontent.com/8952138/109216386-510cd900-7769-11eb-870a-5722c22ebe54.png)

source (feel free to add tests): https://roamresearch.com/#/app/athensresearch/page/kWDwfUWfL

Another case while creating the Roam demo above: block embed in linked refs? Maybe it'll work itself out.

Reply:
This patch is already bloated. Let me take up the parser stuff(this and #668) in the next patch

Reply:
test and behavior decisions

https://www.loom.com/share/a66a953aa915449299954734c7c5a8a5
https://www.loom.com/share/8117e03734ca4c8ba10c49399944b575